health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -0,0 +1,440 @@
+            </div>
+        )
+    }
+    else
+    {
+        return (
+            <React.Fragment>
+                {firstPage && 
+                    <div>
+                        <Card type={"primary"} variant={"viewcard"} className={"example-view-card"}>
+                            <div style={{ fontSize: "2.5rem", fontWeight: 700 }}>{t(`HIERARCHY`)} {hierarchyType}</div>
+                            <div style={{ height: "2rem" }}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)